lib/public_key/src/pubkey_os_cacerts.erl

```diff
@@ -0,0 +1,184 @@
+%%
+%% %CopyrightBegin%
+%%
+%% Copyright Ericsson AB 2008-2022. All Rights Reserved.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%
+%% %CopyrightEnd%
+%%
+
+%%
+
+-module(pubkey_os_cacerts).
+
+-include("public_key.hrl").
+-export([load/0, load/1, get/0]).
+
+-on_load(on_load/0).
+-nifs([os_cacerts/0]).
+
+%% API
+
+%% Return cacerts
+-spec get() -> [public_key:combined_cert()].
+get() ->
+    case persistent_term:get(?MODULE, not_loaded) of
```

Comment:
> If I understand correctly, this persistent_term would stay in memory forever

True, but do we really need to handle that case? 
I.e. add a delete_cacerts() function to the API that will never be called, or do you see another solution.

There is no automated reload, which is what I wanted:

1.   We don't want change the public_key from a library application

2.  I think I implemented the original cache in ssl and Ingela have been fixing bugs in that since then.
     So I want to keep it simple.
    The user (should) know when the system is updated and can thus call **load/[0|1]**